Gopkg.toml

```diff
@@ -1,26 +1,51 @@
-[[constraint]]
+# Force dep to vendor the code generators, which aren't imported just used at dev time.
+required = [
+  "k8s.io/code-generator/cmd/defaulter-gen",
+  "k8s.io/code-generator/cmd/deepcopy-gen",
+  "k8s.io/code-generator/cmd/conversion-gen",
+  "k8s.io/code-generator/cmd/client-gen",
+  "k8s.io/code-generator/cmd/lister-gen",
+  "k8s.io/code-generator/cmd/informer-gen",
+  "k8s.io/code-generator/cmd/openapi-gen",
+  "k8s.io/gengo/args",
+]
+
+[[override]]
+  name = "k8s.io/code-generator"
+  # revision for tag "kubernetes-1.11.4"
+  revision = "8c97d6ab64da020f8b151e9d3ed8af3172f5c390"
+
+[[override]]
   name = "k8s.io/api"
-  version = "kubernetes-1.10.0-beta.2"
+  # revision for tag "kubernetes-1.11.4"
+  revision = "37c5ce6f2f592fbbd798bb86a8814d0918b3abe1"
 
-[[constraint]]
+[[override]]
   name = "k8s.io/apiextensions-apiserver"
-  version = "kubernetes-1.10.0-beta.2"
+  # revision for tag "kubernetes-1.11.4"
+  revision = "67bb9d92aa024a38a3e323cbb9e115e82c0d50b3"
 
-[[constraint]]
+[[override]]
   name = "k8s.io/apimachinery"
-  version = "kubernetes-1.10.0-beta.2"
+  # revision for tag "kubernetes-1.11.4"
+  revision = "8ee1a638bafa4ae9691077e690cb45dd54f45111"
 
-[[constraint]]
+[[override]]
   name = "k8s.io/client-go"
-  version = "kubernetes-1.10.0-beta.2"
+  # revision for tag "kubernetes-1.11.4"
+  revision = "3db8bfc8858dc9a5d6e7ef5817f58a7ca30b0c6a"
+
+[[override]]
+  name = "sigs.k8s.io/controller-runtime"
```

Comment:
I don't think we're using the runtime anywhere in the etcd-operator just yet.
So probably remove this?

Reply:
@hasbro17 thanks, updated.